sdk/python/kfp/compiler/compiler.py

```diff
@@ -194,15 +196,16 @@ def _get_groups_helper(group):
       return groups
 
     return _get_groups_helper(root_group)
-        
+
   def _get_uncommon_ancestors(self, op_groups, op1, op2):
     """Helper function to get unique ancestors between two ops.
 
     For example, op1's ancestor groups are [root, G1, G2, G3, op1], op2's ancestor groups are
     [root, G1, G4, op2], then it returns a tuple ([G2, G3, op1], [G4, op2]).
     """
     both_groups = [op_groups[op1.name], op_groups[op2.name]]
-    common_groups_len = sum(1 for x in zip(*both_groups) if x==(x[0],)*len(x))
+    common_groups_len = sum(
```

Comment:
Can you please undo the auto-format changes so that the diff is smaller and changes are more clear.

Reply:
ack. thanks